assets/js/pages/ChecksCatalog/CatalogContainer.jsx

```diff
@@ -9,6 +9,7 @@ import LoadingBox from '@common/LoadingBox';
 function CatalogContainer({
   onClear = () => {},
   onRefresh = () => {},
+  withResetFilters = true,
```

Comment:
You can set `withResetFilters` to `false` by default and have the API be more idiomatic (and less verbose) whenever you want to set that prop to true

`<CatalogContainer withResetFilters />`